region/maridia/inner-pink/Aqueduct.json

```diff
@@ -2502,6 +2531,83 @@
       ],
       "devNote": "This does not include canRiskPermanentLossOfAccess, as it is only worth doing this strat if the items are there."
     },
+    {
+      "link": [5, 7],
+      "name": "G-Mode Overload PLMs - Power Bomb the Items, Bootless UWJ",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": false
+        }
+      },
+      "requires": [
+        {"notable": "G-Mode Overload PLMs - Power Bomb the Items"},
+        {"or": [
+          {"itemNotCollectedAtNode": 7},
+          {"itemNotCollectedAtNode": 8}
+        ]},
+        "h_usePowerBomb",
+        "h_usePowerBomb",
+        "canSnailClimb",
+        "canBootless2WideUWJ"
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "PLMs can be overloaded in direct G-Mode with a single Power Bomb if both items are still there and 2 PBs if only one item is.",
+        "There is a row of tiles that works, just above and to the left of the right door.",
+        "The row is one tile higher than the horizontal pipe that is part of the door frame, spanning from the left to one tile away from the right wall.",
+        "Climb one snail to place the Power Bomb, then another to reduce the distance of the wall jump climb."
+      ],
+      "devNote": "This does not include canRiskPermanentLossOfAccess, as it is only worth doing this strat if the items are there."
+    },
+    {
+      "link": [5, 7],
+      "name": "G-Mode Morph Overload PLMs - Power Bomb the Items, Bootless UWJ",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": true
+        }
+      },
+      "requires": [
+        {"notable": "G-Mode Overload PLMs - Power Bomb the Items"},
+        {"or": [
+          {"itemNotCollectedAtNode": 7},
+          {"itemNotCollectedAtNode": 8}
+        ]},
+        "h_artificialMorphSpringBall",
+        "h_artificialMorphPowerBomb",
+        "h_artificialMorphPowerBomb",
+        "canSnailClimb",
+        "canBootless2WideUWJ"
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "PLMs can be overloaded in direct G-Mode with a single Power Bomb if both items are still there and 2 PBs if only one item is.",
+        "There is a row of tiles that works, just above and to the left of the right door.",
+        "The row is one tile higher than the horizontal pipe that is part of the door frame, spanning from the left to one tile away from the right wall.",
+        "Climb one snail to place the Power Bomb, then another to reduce the distance of the wall jump climb."
+      ],
+      "devNote": "This does not include canRiskPermanentLossOfAccess, as it is only worth doing this strat if the items are there."
+    },
+    {
+      "link": [5, 7],
+      "name": "G-Mode Morph Overload PLMs - Bomb the Speed Blocks, Bootless UWJ",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
```

Comment:
Why is this special, and direct?

Reply:
This one is separate because its artificial morphed from the right door - if it went down to the G-mode morph junction, it would need Morph, HiJump, or Gravity to snail climb back up.

Fixed the issue with direct, and removed a 10->7 strat, as it could just go 10->11->7 with those same items